### PR TITLE
Add otlhs

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/otlhs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - openturns
+
+test:
+  imports:
+    - otlhs
+
+about:
+  home: http://openturns.org/
+  license: LGPLv3
+  summary: Optimized LHS module
```
Comment (synapticarbors) [inline recipes/otlhs/meta.yaml]: - Add `dev_url: https://github.com/openturns/openturns`
- Maybe add a more in detail `description:` section. Maybe just copy the first paragraph from the README: https://github.com/openturns/openturns#openturns-open-source-initiative-to-treat-uncertainties-risksn-statistics

Comment (jschueller) [inline recipes/otlhs/meta.yaml]: ok, done

File: recipes/otlhs/bld.bat [hunk 2]
```diff
@@ -0,0 +1,2 @@
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
```
Comment (patricksnape) [inline recipes/otlhs/bld.bat]: There's no chance of us building this rather than copying a binary?

File: recipes/otlhs/meta.yaml (hunk 1 above)
Comment (synapticarbors) [inline recipes/otlhs/meta.yaml]: The `description` can be pretty verbose and multi-line. I would literally copy the entire paragraph:

```
    description: |
        OpenTURNS is a scientific C++ and Python library including an internal data
        model and algorithms dedicated to the treatment of uncertainties. The main 
        function of that library is giving to specific applications all the functionalities 
        needed to treat uncertainties in studies. Targeted users are all engineers who 
        want to introduce the probabilistic dimension in their so far deterministic studies.

File: recipes/otlhs/bld.bat (hunk 2 above)
Comment (jschueller) [inline recipes/otlhs/bld.bat]: no, as openturns cannot be compiled with vc compilers

File: recipes/otlhs/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/otlhs/meta.yaml]: it's not openturns, but depends on it, I added a 2 line description.

**Comment by jschueller** [comment]
@patricksnape good to go ?

**Comment by jakirkham** [comment]
We have MinGW compilers via `conda` if that would help.

**Comment by jschueller** [comment]
@jakirkham, I looked at the mingwpy package, nice! I see it's used to build pure a few Python packages. Unfortunately I would need a lot more mingw packages to meet OpenTURNS dependencies (boost, lapack, libxml, nlopt ...) and the dependencies of all of it's modules.

**Comment by jakirkham** [comment]
So @mingwandroid has done a lot of work packaging things from MSYS2. Maybe he could provide you some advice on what packages to use to get a working source build.

**Comment by mingwandroid** [comment]
When I get back from holiday/vacation I will look into these issues.

**Comment by jschueller** [comment]
could this be merged in the mean time ?

**Comment by jakirkham** [comment]
So we already repackage binaries for other OT things. This doesn't seem any different to me. :+1: for merging. What do you think @patricksnape ?

**Comment by patricksnape** [comment]
I'd rather by pragmatic and let people get on with their work. Build where we can and ignore Windows where we have to 😆 LGTM

**Comment by jakirkham** [comment]
Did this feedstock get removed?

**Comment by jschueller** [comment]
Yes, I requested it, it got merged into openturns.

**Comment by jakirkham** [comment]
Ok, have removed the associated GitHub team.

**Pull request merged**